llvm/test/CodeGen/RISCV/GlobalISel/legalizer/rvv/legalize-load.mir

```diff
@@ -0,0 +1,29 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+# RUN: llc -mtriple=riscv32 -mattr=+v -run-pass=legalizer %s -o - | FileCheck %s
+# RUN: llc -mtriple=riscv64 -mattr=+v -run-pass=legalizer %s -o - | FileCheck %s
```

Comment:
You changed G_LOAD and G_STORE legalizer rules above. I only see G_LOAD tests. Either we need to add tests and change PR description or we need to not change legalizer rules for G_STORE.